zo0r/react-native-push-notification
# Fix example app cancel notification

When testing the example app, I found it unable to cancel the last notification.  
This issue was caused by the `lastId` variable representing the next id to be used instead of the id of the last created notification.
Because of that the `cancelNotif` function would try to cancel a notification not yet created.

To test this change:
* Run the example app
* Press `Schedule a notification in 30s`
* Press `Cancel last notification (if any)`
* Check if the scheduled notification was cancelled
